tracing/opentracing/client_interceptors.go

```diff
@@ -113,6 +114,8 @@ func newClientSpanFromContext(ctx context.Context, tracer opentracing.Tracer, fu
 	if parent := opentracing.SpanFromContext(ctx); parent != nil {
 		parentSpanCtx = parent.Context()
 	}
+	// Inject tags for the first time
+	ctx = grpc_ctxtags.NewTagsForCtx(ctx)
```

Comment:
I think it would be better if this injected ctx tags if one already exists in context it shouldn't be implicitly creating one.

Reply:
I think we always want to have ctx_tags interceptor before tracing / logging etc. this would mean the other interceptor has already set up the context for tags.

Reply:
AFAIK this method is also used to create a tag inside the context for the very first time. How would we initialise a tag otherwise?

Reply:
I think we dont need this as `grpc_tags` should always be the first thing in the interceptor chain and will set up tags etc. before the tracing / logging middleware is involked.

Reply:
@domgreen You are right for the server side interceptor, but there isn't a client interceptor what injects the `grpc_tags` into the request context

Reply:
Discussed with @domgreen offline, will create a client interceptor for making sure that the tag map is present in the context.